gubernator/templates/pr_dashboard.html

```diff
@@ -22,6 +22,7 @@
 		% continue
 	% endif
 	<h3 class="pr-section">
+	<a id="section-{{title|slugify}}" />
```

Comment:
I didn't know that. I tested locally in Chrome and it worked, but I guess other browsers have different behaviours? Either way, better to play it safe and fixed it.